src/renderers/shaders/ShaderChunk/envmap_pars_fragment.glsl

```diff
@@ -4,6 +4,11 @@
 #endif
 
 #ifdef USE_ENVMAP
+
+	#if defined( USE_BUMPMAP ) || defined( USE_NORMALMAP ) || defined( PHONG )
```

Comment:
```glsl
#if ! defined( STANDARD ) && ( the rest )
```

will make it fully equivalent, probably silence the warning, and therefore re-add the problem (currently present on HEAD) that standard material may actually try to use `vWorldPosition` without ever declaring the varying, if I'm not mistaken.